app/models/mixtape.rb

```diff
@@ -0,0 +1,10 @@
+class Mixtape
+  include ActiveModel::SerializerSupport
+
+  attr_reader :songs
+
+  def initialize(genre:,songs:)
```

Comment:
You might gain `ActiveModel::SerializerSupport` by default (though not 100% on that).